Source/SwiftLintFramework/Rules/Idiomatic/DiscouragedObjectLiteralRule.swift

```diff
@@ -1,4 +1,4 @@
-public struct DiscouragedObjectLiteralRule: ASTRule, OptInRule, ConfigurationProviderRule {
+public struct DiscouragedObjectLiteralRule: ASTRule, OptInRule, ConfigurationProviderRule, ManuallyTestedExamplesRule {
```

Comment:
Seems like this one could remove this conformance if `DiscouragedObjectLiteralRuleTests.swift` was renamed to `DiscouragedObjectLiteralRuleConfigurationTests.swift` like you did with the other rules in #4061.

Reply:
I have already prepared another follow-up to do exactly that.